app/models/spree/admin/main_menu/root.rb

```diff
@@ -7,10 +7,6 @@ class Root < Section
         def initialize
           super('root', 'root', nil, nil, [])
         end
-
-        def add_to_section(section_key, item)
```

Comment:
This hasn't been used anywhere. Moreover, `add_item` is not defined anywhere. Even if it's a typo, and should be `add_item` the whole does what `add` method (defined in `Section` class) does.

Reply:
It's not used in our code, because it's a method for extension developers who need to add a custom item to the admin panel main menu. That's not a reason to delete it

Reply:
I may have missed a test for this one, but let's just fix the implementation

Reply:
I've added it back, and wrote a test for it.

Reply:
Perfect, thanks!